packages/core/src/contrib/Functions.ts

```diff
@@ -2026,10 +3042,10 @@ const closestPointEllipseCoords = (
 
 // `_compDictVals` causes TypeScript to enforce that every function in
 // `compDict` takes a `Context` as its first parameter and returns a `Value`
-const _compDictVals: ((
-  context: Context,
-  ...rest: never[]
-) => Value<ad.Num>)[] = Object.values(compDict);
+// const _compDictVals: ((
+//   context: Context,
+//   ...rest: never[]
+// ) => Value<ad.Num>)[] = Object.values(compDict);
```

Comment:
I'd delete this code instead of commenting it out, but I actually think you should uncomment it instead (see the comment I left above about precise types).